docs/_source/tutorials/notebooks/training-llm-mistral-sft.ipynb

```diff
@@ -0,0 +1,602 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ⓜ️ Finetuning LLMs: Supervised Finetuning on Mistral 7B\n",
```

Comment:
maybe emphasize it is chat based in the title too?

Reply:
Great idea, handled in 3d221adbae61bcf23ca8aee740a31f678a6df230